_includes/footer.html

```diff
@@ -50,6 +50,9 @@ <h1 class="title">Get Involved</h1>
               <li>
                 <a href="https://www.grin-forum.org/"><i class="fab fa-discourse"></i>Forum</a>
               </li>
+              <li>
+                <a href="https://discord.gg/Z3sEfEU"><i class="fab fa-discord"></i>Discord</a>
```

Comment:
Why is there a separate discord and grin-forum is not enough?

Reply:
seperate discord? and people could choose between the forum and discord, i personally prefer discord